requirements/tests.in

```diff
@@ -0,0 +1,8 @@
+-r base.txt
+
+behave
+coverage
+flake8-tuple
+flake8==3.7.9
+pytest-random-order
+factory_boy
```

Comment:
I'll be using most of these in the myco tests I'm working on, this is why I added them in this PR

Reply:
> General comments:
> 
> * some of the tox environments are currently not working (e.g. `check_readme` and `coverage`). I think it would be good to either fix them or remove them.
> * In this PR you added some dependencies because you want to use them in another PR. However, this will create an inconsistency in the repo history, because if I'll want to understand when/why a dependency was introduced I will find this unrelated PR instead. It's ok for now, but I'd recommend avoiding it in the future.
> * At the moment, the `tox.in` file is using both tabs and spaces for indentation. Mixing these two styles is not recommended and it shows on the diff, creating inconsistencies. It would be good to convert all tabs to spaces.

Thanks for the review, atm the check_readme test is working and displaying problems with the readme file which I won't target in this PR
I have also commented out the unittests environment for now as there are no unit tests and this was causing erros